example/schemavalidator/schemavalidator.cpp

```diff
@@ -7,9 +7,124 @@
 #include "rapidjson/schema.h"
 #include "rapidjson/stringbuffer.h"
 #include "rapidjson/prettywriter.h"
+#include <string>
+#include <iostream>
 
 using namespace rapidjson;
 
+typedef GenericValue<UTF8<>, CrtAllocator > ValueType;
+
+// Forward ref
+static void CreateErrorMessages(const ValueType& errors, size_t depth, const char* context);
+
+// Convert GenericValue to std::string
+static std::string GetString(const ValueType& val) {
+    std::string str("");
```

Comment:
How about just use Writer to output a JSON string for this conversion?

Reply:
`to_string()` is C++11 as well.

Reply:
@miloyip I didn't use Writer because this is not for writing JSON value. I just want the stringified value, for use as message insert. Writer will add double quotes for String type. Please show how you would prefer to fix this.

Reply:
Writer outputs JSON values, so strings are quoted, which is not what I want here. I have used ostringstream instead. (It would be nice if GenericValue had a to_string() method :) )